components/admin_console/billing/billing_subscriptions.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import FormattedAdminHeader from 'components/widgets/admin_console/formatted_admin_header';
+
+import './billing_subscriptions.scss';
+
+type Props = {
+
+};
+
+const BillingSubscriptions: React.FC<Props> = () => {
```

Comment:
Recommend removing `Props` here if they're empty.

```suggestion
const BillingSubscriptions: React.FC = () => {
```